lib/src/modules/inventoryItem/InventoryItem.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import cx from 'classnames';
+import { bool, string } from 'prop-types';
+import { ButtonIconContained, TooltipWrapper } from 'lib';
+
+export function InventoryItem({
+  isDragging,
```

Comment:
🟢  Feels a little strange how we have the dragging props here. I believe it's just the name of them, is there a more abstracted naming convention we could use?

For example, `isDragPreview` could just be `preview` to say "this is the component when it's previewing the inventory" - I don't think we need to be specific about the interaction.

Reply:
This is fair, but its a bit of a weird one that I went back and forth on when making this! While I know that this is a public repository, realistically we're the ones using it and those styles are used when those things are happening. And this is the name of what we pass down from a `Draggable` so it's not a completely alien concept to GUI.

Reply:
I've changed `isDragPreview` to just `isPreview`, but I can't think of a vaguer name for the `isDragging` part! 

Maybe now we're using `Draggable` in the story that links the two together anymore so the prop name doesnt feel so out of place?